Automattic/wp-calypso
# [Odie] Make Wapuu chat as the default route

Related to https://github.com/Automattic/wp-calypso/pull/81645

## Proposed Changes

It completes the changes for setting the Wapuu chat as the main thing to see when opening the Help Center.

## Testing Instructions

### Regression Test

A smoke test of the Help Center should suffice. Eg, opening/closing/minimising the help center, and navigating a bit through it.

### Test

Add the feature flags "wapuu" and assert the following is true.
1. When opening the Help Center, the first thing to see is the the suggested links
2. When clicking on "Still need help?" Wapuu chat bot is presented.
3. Navigate anywhere and check that minimise, check that the it will keep the state when restoring
4. Navigate back button will take you to the main Help Center section.

Here I don't know what would be the best UX. if we open the Help Center with Wapuu as the first thing to see, clicking back should take you to the Contact Options or to the main Help Center screen

## Pre-merge Checklist
